### PR TITLE
Added enable/ disable logging api in Manager class

### PR DESCRIPTION
**Pull request by PramodKAvvaru**

### CODE REVIEW

File: netconf_client/ncclient.py [hunk 1]
```diff
@@ -104,6 +104,12 @@ def logger():
         """Returns the internally used logger instance (same for all sessions)"""
         return _logger
 
+    def enable_logging(self, level=logging.DEBUG):
+        _logger.setLevel(level)
+
+    def disable_logging(self, level=logging.NOTSET):
+        _logger.setLevel(level)
+
```
Comment (Nitori-) [inline netconf_client/ncclient.py:112]: From what I'm seeing in the docs, using `logging.NOTSET` doesn't actually disable logging per-se.

How do you feel about this instead?

```suggestion
    def set_logger_level(self, level):
        _logger.setLevel(level)

```

**Review by Nitori- — COMMENTED** [review]
Very sorry for the delay! I seem to have missed the notification for this pull request.

File: netconf_client/ncclient.py (hunk 1 above)
Comment (PramodKAvvaru) [reply netconf_client/ncclient.py:112]: that looks good to me 👍

**Review by Nitori- — APPROVED** [review]

**Comment by Nitori-** [comment]
Thanks for the contribution! We should get this out in the next release. I'll try to bundle it with the changes from #9 in the next few days.

**Pull request merged**